### PR TITLE
Zijie_Yu finished HW1

### PR DESCRIPTION
**Pull request by iTvX**
Name: Zijie Yu

All content in hw1.md, and assets file is for the pic.

### CODE REVIEW

File: ShortQuestions/hw1/hw1.md
Comment (connorwang0901) [inline ShortQuestions/hw1/hw1.md:1]: I  like when you describe the life cycle of Maven, they are detailed and clear, thanks!

Comment (fzybb) [reply ShortQuestions/hw1/hw1.md:1]: We were friends on WeChat 3 years ago lol

**Comment by 1109122780** [comment]
Great Work！

**Review by 1109122780 — COMMENTED** [review]
Great Work!

File: ShortQuestions/hw1/hw1.md
Comment (1109122780) [reply ShortQuestions/hw1/hw1.md:1]: lol

Comment (iTvX) [reply ShortQuestions/hw1/hw1.md:1]: > I like when you describe the life cycle of Maven, they are detailed and clear, thanks!

Thanks~

**Comment by LeiChenUCD** [comment]
good job

**Pull request merged**